src/v/transform/rpc/deps.cc

```diff
@@ -188,13 +188,18 @@ class partition_manager_impl final : public partition_manager {
             co_return response;
         }
         auto stm = partition->transform_offsets_stm();
-        auto result = co_await stm->get(request.key);
-        if (!result) {
-            response.errc = result.error();
-            co_return response;
-        }
         response.errc = cluster::errc::success;
-        response.result = result.value();
+        for (auto key : request.keys) {
+            auto result = co_await stm->get(key);
+            if (!result) {
+                response.errc = result.error();
```

Comment:
does this mean no partial failure on the request?

Reply:
Yes, do you think it's worth supporting partial failure?

Reply:
I don't think so, it's probably a lot more trouble than its worth.

Reply:
I guess the one case would be if you have a problematic key, would it ever block progress?

Reply:
yes it would block progress because new transforms with that key in the batch would fail... I'll make the failures per key.

Reply:
Done - I've also fixed this for coordinator lookups in addition to commit lookups